recipe/win64_detect_conda.diff

```diff
@@ -0,0 +1,18 @@
+diff --git a/cupy/_environment.py b/cupy/_environment.py
+index efc02b4ce..8c2ddebee 100644
+--- a/cupy/_environment.py
++++ b/cupy/_environment.py
+@@ -202,7 +202,12 @@ def _setup_win32_dll_directory():
+         # Path to the CUDA Toolkit binaries
+         cuda_path = get_cuda_path()
+         if cuda_path is not None:
+-            cuda_bin_path = os.path.join(cuda_path, 'bin')
++            is_conda = ((os.environ.get('CONDA_PREFIX') is not None)
++                        or (os.environ.get('CONDA_BUILD_STATE') is not None))
++            if is_conda:
++                cuda_bin_path = cuda_path
```

Comment:
TODO: think about `$CONDA_PREFIX\Library\bin`

Reply:
How about instead patching `cupy._environment._get_cuda_path()`  code to use `CONDA_PREFIX\Library` as a fallback?

Reply:
How does it work? Set `cuda_bin_path` to `$CONDA_PREFIX\Library\bin`? I am thinking it actually has no harm to add both `cuda_path\bin` and `cuda_path\Library\bin` if they both exist, what do you think @kmaehashi?

Reply:
(the benefit of adding both is we don't have to special-case for Conda.)